docs/sources/fundamentals/intro-to-prometheus/index.md

```diff
@@ -0,0 +1,110 @@
+
+Although this view provides some insight into the performance of the system, it doesn’t provide the full story. For a clearer picture of system performance, it is also important to understand the rate of change that shows _how fast the data being written is changing_. To properly monitor disk performance, it is important to see spikes in activity to understand if and when the system is under load, and whether or not disk performance is at risk. PromQL includes a [rate()](https://prometheus.io/docs/prometheus/latest/querying/functions/#rate) function that shows the per-second average rate of increase over `5m` (5-minute) intervals. This view provides a much better idea of what’s happening with the system.
+
+{{< figure src="/media/docs/grafana/intro-prometheus/rate-function.png" max-width="750px" caption="Prometheus rate function" >}}
+
+A counter metric is just one type of metric. Prometheus [supports several others](https://prometheus.io/docs/concepts/metric_types/). A counter is a number (such as total bytes written) that only goes up. A second metric type is called a `gauge`, which can go up or down.
+
+The following gauge visualization shows the total RAM usage on a computer.
+
+{{< figure src="/media/docs/grafana/intro-prometheus/gauge-example.png" max-width="750px" caption="Gauge visualization" >}}
+
+The third metric type is called a `histogram`, which counts observations and puts them into configurable groups. The following example shows floating-point numbers grouped into ranges that show how often each occurred.
+
+{{< figure src="/media/docs/grafana/intro-prometheus/histogram-example.png" max-width="750px" caption="Historgram visualization" >}}
+
+These core concepts of time series, metrics, labels, and aggregation functions are foundational to Grafana and observability.
+
+## Why this is valuable
+
+Software and systems are a difficult business. Sometimes things go wrong. Observability helps you understand a system’s state so that future issues can be proactively addressed. And when problems do occur, you can diagnose and solve them within your Service Level Objectives (SLOs). The [three pillars of observability](https://www.oreilly.com/library/view/distributed-systems-observability/9781492033431/ch04.html) are metrics, logs, and traces and Prometheus supports the metric pillar. When software on a computer runs slowly, observability can help identify whether the CPU is saturated, the system is out of memory, or if the disk is writing at maximum speed.
```

Comment:
no change needed -- but ideally there would be a good reference link on our site to the 3 pillars of observability but there isn't yet a good fit.  Maybe a thing for another day.